getting_started/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -194,3 +207,36 @@ Constants
 
 Use CONSTANT\_CASE, all caps, with an underscore (\_) to separate words:
 ``const MAX_SPEED = 200``
+
+Static Typing
+-------------
+
+Since Godot 3.1, GDScript supports :ref:`optional static typing<_doc_gdscript_static_typing>`.
+
+Type hints
+~~~~~~~~~~
+
+Place the colon right after the variable's name, without a space, and let the GDScript compiler infer the variable's type when possible.
+
+**Good**:
+
+::
+
+   onready var health_bar: ProgressBar = get_node("UI/LifeBar")
+
+   var health: = 0 # The compiler will use the int type
+
+**Bad**:
+
+::
+
+   onready var health_bar: = get_node("UI/LifeBar") # The compiler can't infer the exact type and will use Node instead of ProgressBar
+
+   var health : int = 0
+
+
+Add a space on either sides of the return type arrow when defining functions.
+
+::
+
+   func heal(amount:int) -> void:
```

Comment:
Note, this is different from [what you said earlier](https://github.com/godotengine/godot-docs/issues/2506#issuecomment-500108259), is this intentional?

Reply:
Indeed, should be `amount: int`.